lib/ui/common/ProfileAvatar.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:farmsmart_flutter/model/bloc/ViewModelProvider.dart';
+import 'package:farmsmart_flutter/ui/profile/Profile.dart';
+import 'package:flutter/material.dart';
+
+import 'ViewModelProviderBuilder.dart';
+import 'image_provider_view.dart';
+
+class ProfileAvatar extends StatelessWidget {
+  final ViewModelProvider<ProfileViewModel> _viewModelProvider;
+  final double _width;
+  final double _height;
+  final Color _backgroundColor;
+  final TextStyle _textStyle;
+
+  const ProfileAvatar({Key key,ViewModelProvider<ProfileViewModel> viewModelProvider, double width, double height, TextStyle textStyle, Color backgroundColor}) : this._viewModelProvider = viewModelProvider, this._width = width, this._height=height, this._textStyle= textStyle,this._backgroundColor = backgroundColor, super(key: key);
```

Comment:
As above, this is very difficult to read, just apply the format 👍

Reply:
sorted 👍